mu/interface.py

```diff
@@ -850,6 +850,19 @@ def enable(self, sibling):
         self.setAcceptDrops(True)
         sibling.setAcceptDrops(True)
 
+    def show_confirm_overwrite_dialog(self):
+        msg = QMessageBox()
```

Comment:
#128 was an odd one and I believe this issue would reemerge here. passing `self` to `QMessageBox()` should avoid it.